R/covr.R

```diff
@@ -222,6 +222,25 @@ package_coverage <- function(path = ".",
 
   flags <- getOption("covr.flags")
 
+  ## BEGIN Oracle Contribution
+  # License: GPL-3 with additional permission for MIT under GPL-3 Section 7 as set forth in license documents.
+  #
+  # check for icc compiler
+  compiler <- get_compiler()
+  if (compiler == "icc")
```

Comment:
This should follow the conventions in the package and use [1TBS](https://en.wikipedia.org/wiki/Indent_style#1TBS)